src/main/target/SPEEDYBEEF4/target.c

```diff
@@ -0,0 +1,43 @@
+/*
+* This file is part of Cleanflight.
+*
+* Cleanflight is free software: you can redistribute it and/or modify
+* it under the terms of the GNU General Public License as published by
+* the Free Software Foundation, either version 3 of the License, or
+* (at your option) any later version.
+*
+* Cleanflight is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+* GNU General Public License for more details.
+*
+* You should have received a copy of the GNU General Public License
+* along with Cleanflight.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#include <stdint.h>
+
+#include <platform.h>
+#include "drivers/io.h"
+
+#include "drivers/dma.h"
+#include "drivers/timer.h"
+#include "drivers/timer_def.h"
+
+const timerHardware_t timerHardware[USABLE_TIMER_CHANNEL_COUNT] = {
+    DEF_TIM(TIM5, CH4, PC11,  TIM_USE_PPM,   0, 0), // PPM
+
+    DEF_TIM(TIM8, CH1, PC6,  TIM_USE_MOTOR, 0, 0), // S1  UP(1,2)
+    DEF_TIM(TIM8, CH2, PC7,  TIM_USE_MOTOR, 0, 1), // S2  UP(2,1)
+    DEF_TIM(TIM8, CH3, PC8,  TIM_USE_MOTOR, 0, 1), // S3  UP(2,1)
```

Comment:
TIM_USE_MOTOR is not valid. You need TIM_USE_MC_MOTOR for a multirotor motor, TIM_USE_FW_MOTOR for a fixed wing motor or TIM_USE_FW_SERVO for fixed wing servo.